app/src/main/java/fr/free/nrw/commons/contributions/ContributionsFragment.java

```diff
@@ -572,16 +572,17 @@ private void fetchCampaigns() {
         if (store.getBoolean("displayCampaignsCardView", true)) {
             presenter.getCampaigns();
         }
+        else{
+            campaignView.setVisibility(View.GONE);
+        }
     }
 
     @Override public void showMessage(String message) {
         Toast.makeText(getContext(), message, Toast.LENGTH_SHORT).show();
     }
 
     @Override public void showCampaigns(Campaign campaign) {
-        if (campaign != null) {
```

Comment:
The PR seems right in general, thanks @4D17Y4 :) But why did you removed this null check? I think it should stay.

Reply:
Actually the null chek is handled on function call below. It sets the visibility of the campaign view to gone if the campaign is null. So I don't think we need it.

Reply:
https://github.com/commons-app/apps-android-commons/blob/ac297e42f601e88f1fe24969822cb76762a13e8b/app/src/main/java/fr/free/nrw/commons/campaigns/CampaignView.java#L48-L56

This is the mentioned method.
Also, if the campaign is null the condition check will bypass the functions check and the campaignView visibility would still be visible which may cause some unexpected errors.

I was on another device so could not reference the code in the above comment. ;)

Reply:
For now, you are right it seems redundant. However, I think having the null check inside the method will be a better practice. Some people may use showCampaign method without nullcheck in future. So I think we should keep it inside the method :)

Reply:
I get it @neslihanturan,
Sorry for trouble,
I've made the required changes
Thanks.